dashboard/modules/log/log_agent.py

```diff
@@ -337,12 +337,14 @@ async def _find_task_log_offsets(
         )
 
         if task_attempt_magic_line_offset == -1:
-            raise FileNotFoundError(
-                f"Log for task attempt({task_id},{attempt_number}) not found"
+            # TODO(rickyx): This means the token is missing in the corresponding
```

Comment:
https://github.com/ray-project/ray/blob/736060e7344cbfcb5105541baa6e6da1fe72857a/python/ray/tests/test_state_api_log.py#L1303-L1328 

Test that covers concurrent actors